### ISSUE TITLE
Tab to switch language is broken in some places

### ISSUE THREAD

**Issue by TerryCavanagh** [TerryCavanagh/diceydungeons.com#735]
It doesn't work for character description text, quest descriptions, or here:

![dice](https://user-images.githubusercontent.com/2874150/50710894-35638780-1064-11e9-87e2-2bad15c29bf1.png)

```
as you can see, the "current gadgets" does not switch language when pressing tab. (also the French translation of "acceleration" is wrong, it says "All counters to 6")
```

**Comment by TerryCavanagh** [comment]
Huh. Weirdly, I can't replicate this one? Fixed the quest and character description ones, though.

**Comment by TerryCavanagh** [comment]
We've pretty solidly fixed this one in v1.9!
